cwl/src/main/scala/cwl/ExpressionEvaluator.scala

```diff
@@ -60,6 +71,28 @@ object ExpressionEvaluator {
     }
   }
 
+  def evalInterpolatedString(string: InterpolatedString, parameterContext: ParameterContext, expressionLib: ExpressionLib): ErrorOr[WomValue] = {
+    def interpolate(remaining: String, acc: String = ""): ErrorOr[String] = {
```

Comment:
ToL I'm glad this works so it's ToL, but it does feel a little like re-inventing the wheel vs. using a parser library.  It would potentially be less code, arguably easier to maintain

Reply:
tbh I don't think I could write this using a parser library in a justifiable amount of time. 😢